<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>161</width>
    <height>235</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame { background: transparent; border: none; } 

QPushButton {
border: none; 
	color: white; 
	background-color: black; 
	font: bold large &quot;Rockwell Extra Bold&quot;; 
	font-size: 18px; 
}

QPushButton:pressed {
		color: gold; 
}
QPushButton:hover {
		color: gold; 
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="m_newGameBtn">
      <property name="text">
       <string>Play Game</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_settingsBtn">
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_aboutBtn">
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_quitBtn">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
